README.md

```diff
@@ -58,103 +61,151 @@ then accepted by a browser if the set conforms to its policy.
+*   Site authors must submit their First-Party Set declarations for acceptance (see [UA Policy](#ua-policy) for proposed documented criteria).
+*   Sets will expire after a prescribed period of time, and be required to undergo renewal. This prevents sets from becoming stale, in case domain ownership changes.
+*   Each set is indicated by the owner site, and member sites. 
+
+    ```
+    { owner: "https://fps-owner.example", 
+      members: ["https://fps-member1.example",
+      "https://fps-member2.example"]}
+
+    ```
 
-Web platform features should _not_ use first-party sets to make one origin's state directly accessible
-to another origin in the set. First-party sets should only control when embedded content can access its own state.
-That is, if `a.example` and `b.example` are in the same first-party set, the same-origin policy
-should still prevent `https://a.example` from accessing `https://b.example`'s IndexedDB databases.
-However, it may be reasonable to allow a `https://b.example` iframe within `https://a.example` to
-access the `https://b.example` databases.
+Technical consistency and freshness checks must be performed on the list:
 
-# Design details
+*   No domain can appear in more than one set.
+*   Expired sets must be removed.
+
+Static lists are easy to reason about and easy for others to inspect. At the same time, they can develop deployment and scalability issues. Changes to the list must be pushed to each user's browser via some update mechanism. This complicates sites' ability to deploy new related domains, particularly in markets where network connectivity limits update frequency. They also scale poorly if the list gets too large. When such considerations outweigh the benefits of the lower implementation complexity of static lists, browsers may consider using the [Signed Assertions based design](signed_assertions.md) that proposes fetching sets from a `.well-known` location of the website.
+
+
+# Acceptance Process
+
+This section proposes a possible model for a First-Party Set acceptance process that could be shared across all browsers. However, many aspects of the process and policy will need to be tuned based on feedback from the web ecosystem.
+
+## Submission
+
+Sites will need to submit their proposed group of domains to a public tracker (such as a dedicated GitHub repository, like that of the [Public Suffix List](https://github.com/publicsuffix/list/wiki/Guidelines), and [Disconnect’s entities list](https://github.com/disconnectme/disconnect-tracking-protection/issues?q=is%3Aissue+%22entity%22+)), along with information needed to satisfy the UA policy. Technical verification of the submitter’s control over the domains may also require a challenge to be served at a `.well-known` location on each of the domains in the set.
```

Comment:
Great to see expansion on this, but I'm still concerned about how this would work in practice. I acknowledge that it's (rightfully) still very open and subject to evolution. But I'm imagining either:

* there is one list that all UAs have come to agreement on policy on, and they all pull from the same list. I see the practicality of this, but it's dependent on UAs agreeing policy (not a guarantee) and for something with such a broad scope (any site on the web can play) feels far too much centralisation.
* sites have to submit their set for approval to *every* UA, which doesn't seem realistic. Even with just mainstream UAs right now, assuming they all implement FPS, I can see site admins submitting requests to MS, Apple, Mozilla and Google, but not bothering with Opera, Samsung, let alone Vivaldi, Brave, Tor... And what about mobile browsers? Whether or not requests need to be submitted to eg. Firefox mobile separately from Firefox desktop may not be obvious to site admins.
* the middle ground: some UAs agree policy and share a list, other UAs might fork the list, others might use the list but take liberties with how they implement it (eg. customise it). Besides fragmentation issues, this becomes confusing for site owners with regards to knowing where to submit their sets for verification.

Basically I'm just struggling to imagine how the verification process to determine whether sites are legitimately in the same set can reasonably scale to the whole web with this level of centralisation, and without [automation](https://github.com/privacycg/first-party-sets/issues/43).

Reply:
Thanks for the feedback, Amy. Just acknowledging that I saw this. We're discussing the [technical-only enforcement issue](https://github.com/privacycg/first-party-sets/issues/43) that you linked to within PrivacyCG, and also plan to have another conversation on the policy aspects. I'll come back and address this once I hear from the others as well.

Reply:
Amy, FYI, I opened issue #48 to keep track of this feedback.